src/core/stubs.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
Is this file here because it's expected by the lib? I mean instead of placing this on literals.

Reply:
nvm, I saw the comment on the other PR